lib/kubernetes-deploy/container_logs.rb

```diff
@@ -39,17 +41,21 @@ def print_all
 
     private
 
-    def fetch_latest(kubectl)
+    def fetch_latest
       cmd = ["logs", @parent_id, "--container=#{container_name}", "--timestamps"]
       cmd << if @last_timestamp.present?
-        "--since-time=#{rfc3339_timestamp(@last_timestamp)}"
-      else
-        "--tail=#{DEFAULT_LINE_LIMIT}"
-      end
+               "--since-time=#{rfc3339_timestamp(@last_timestamp)}"
+             else
+               "--tail=#{DEFAULT_LINE_LIMIT}"
+             end
       out, _err, _st = kubectl.run(*cmd, log_failure: false)
       out.split("\n")
     end
 
+    def kubectl
```

Comment:
It isn't the greatest to have to give this class permanent `kubectl` access, but I think it's acceptable (unlike doing so in `KubernetesResource` for instance).